Week-7/bestTime.js

```diff
@@ -0,0 +1,34 @@
+// Problem 6.4 : Best time to buy and sell stock
+// You are given an array prices where prices[i] is the price of a given stock on the ith day.
+// You want to maximize your profit by choosing a single day to buy one stock andchoosing a
+// different day in the future to sell that stock.
+// Return the maximum profit youcan achieve from this transaction.
+// If you cannot achieve any profit, return 0.
+// Example 1: Input: prices = [7,1,5,3,6,4] Output: 5
+// Explanation: Buy on day 2 (price = 1)and sell on day 5 (price = 6), profit = 6-1 = 5.
+// Note that buying on day 2 and selling onday 1 is not allowed because you must buy before you sell.
+
+const stockPrices = [2, 1, 2, 0, 1];
+// [7, 1, 5, 3, 6, 4];
+
+const getMaxProfit = (arr) => {
```

Comment:
add time and space complexity for all the problems

Reply:
@anilrao316 I have added the time and space complexity at the last of each solution. please let me know if there is anything else

Reply:
@anilrao316 any update?

Reply:
looks good!